assets/app/views/_triggers.html

```diff
@@ -1,6 +1,10 @@
 <div class="builds-block">
   <div class="builds" ng-repeat="trigger in triggers">
-    <div ng-repeat="build in trigger.builds" ng-if="build.status.phase == 'New' || build.status.phase == 'Pending' || build.status.phase == 'Running' || (build.metadata.creationTimestamp | ageLessThan : 5 : 'minutes')" class="build animate-repeat osc-object" kind="Build" resource="build">
+    <div ng-repeat="build in trigger.builds"
```

Comment:
in general, I'd be fine with adding `track by (obj | uid)` to any ng-repeat loop